### ISSUE TITLE
[Proposal] Less verbose make:migration input

### ISSUE THREAD

**Issue by lorisleiva** [laravel/ideas#1165]
Hi 👋,

Currently, when generating a migration that creates a new table I write :

```bash
php artisan make:migration create_subscriptions_table --create=subscriptions
```

1. Since it always follows the same convention, i.e. `create_$tableName_table`, I thought it could be a good idea to **provide this convention as a default migration name** when the `name` argument is omitted and the `create` option is provided.
2. We could also decide on a **good default convention for table updates** when only the `table` option is provided.
3. Whilst updating this command, we could also give the options `create` and `table`, the respective shortcuts `-c` and `-t`.

> **Note:** it goes without saying that if the user provides a `name` argument, the command behaves the same as it currently does. So no breaking changes here.

The bash command above could then be shorten to:

```bash
php artisan make:migration --create=subscriptions
php artisan make:migration -c subscriptions
# Both generate a migration named: create_subscriptions_table
```

If you guys, girls and agender people think it's a good idea, I am more than happy to open a PR.
# 💪💾

**Comment by sisve** [comment]
Two of the suggestions are reasonable default values for the migration name in case it isn't provided, and one is to mangle the parameter names into something unreadable.

I'm +2/3 to this suggestion as it is currently written. Do we really have to add short parameter names?

**Comment by fletch3555** [comment]
Agreed.  I don't see a downside to shorthand parameter names, SO LONG AS the long-form continue to exist as well.  Laravel is full of short-hand features for those that prefer it, but nobody is forced to use them.

**Comment by joshuadegier** [comment]
I did a setup for exactly this a few minutes ago, wasn't aware of this `laravel/ideas` repository. I stumbled upon this idea aswel, felt so double in cases where you create a table to name things twice when there's a perfect solution to use a standardized name for it.

You can find it here: https://github.com/PendoNL/framework/blob/5.6/src/Illuminate/Database/Console/Migrations/MigrateMakeCommand.php

**Comment by lorisleiva** [comment]
@joshuadegier Nice, thanks for sharing. I totally agree with your opinion on the updating convention. The issue is that if we provide a default name for `--create`, then `name` will be an optional parameter. Thus, we have to also provide some default for `--table`. And like you said `update_$tablename_table` is just too vague and basically not something that people use.

**Comment by joshuadegier** [comment]
In my code there's only an exception thrown if the parameter is missing when using any other than `--create`. Seems a fair solution, I'll just leave it here to gather some more thoughts of others.
